src/Analysis/GWASUIApp/CaseControlGWAS.jsx

```diff
@@ -567,7 +580,9 @@ const CaseControlGWAS = (props) => {
         const submitEndpoint = `${gwasWorkflowPath}submit`;
         const requestBody = {
             n_pcs: numOfPC,
+            // TODO - deprecated - remove this "covariates" item as selectedCovariates are already part of "variables" below:
             covariates: selectedCovariates.map((val) => val.prefixed_concept_id),
+            variables: getVariablesList(selectedDichotomousCovariates, selectedCovariates),
```

Comment:
yeah I do like this approach I think... we could drop the `covariates` as we mentioned today

Reply:
yeah agreed. keeping state variables separate for these 2 and adding `getVariablesList()` to request is better since we modify (add/delete) those types of covs differently